WADSuite/README.md

```diff
@@ -0,0 +1,44 @@
+# UWPControls
+
+UWPControls is a collection of test scenarios that covers basic interactions with [Universal Windows Platform controls (UI elements)](https://docs.microsoft.com/en-us/windows/uwp/controls-and-patterns/). Use this as a reference on how to interact with certain UWP controls you have in your application.
+
+The test scenarios are written against [AppUIBasics](../../ApplicationUnderTests/AppUIBasics) application that contains all UWP basic controls. This application needs to be installed once in the machine you are running your test. Follow the instruction on the application [README](../../ApplicationUnderTests/AppUIBasics/README.md).
+
+
+## Requirements
+
+- Windows 10 PC with the latest Windows 10 version (Version 1607 or later)
+- Microsoft Visual Studio 2015 or later
+- [AppUIBasics](../../ApplicationUnderTests/AppUIBasics) application
+
+
+## Getting Started
```

Comment:
It'll be helpful to include some notes about installing and running WinAppDriver, as doing so is a required step to run the tests.